src/test/lib/ApiTest.cpp

```diff
@@ -3512,7 +3522,7 @@ void QuicTestConnectionParam()
             TestScopeLogger LogScope1("GetParam");
             MsQuicConnection Connection(Registration);
             TEST_QUIC_SUCCEEDED(Connection.GetInitStatus());
-            SimpleGetParamTest(Connection.Handle, QUIC_PARAM_CONN_SETTINGS, sizeof(QUIC_SETTINGS), nullptr);
+            SimpleGetParamTest(Connection.Handle, QUIC_PARAM_CONN_SETTINGS, sizeof(QUIC_SETTINGS), nullptr, true);
```

Comment:
I believe every usage of `SimpleGetParamTest` with a `QUIC_PARAM_*_SETTINGS` needs to pass `true` now.

Reply:
Including `QUIC_PARAM_GLOBAL_SETTINGS`.